src/test/java/de/retest/web/RecheckSeleniumAdapterTest.java

```diff
@@ -58,6 +59,24 @@ void convertToPeers_should_result_in_valid_tree() throws Exception {
 		assertThat( diffs ).isEmpty();
 	}
 
+	@Test
+	void outline_should_be_added_to_ignored_attributes() {
```

Comment:
Sorry, forgot to say that this test needs `ResourceLock( value = SYSTEM_PROPERTIES, mode = READ_WRITE )`, whereas the one below needs `ResourceLock( value = SYSTEM_PROPERTIES, mode = READ )`. This is because we parallelize them. (See https://github.com/retest/recheck-web/pull/83.)